### DISCUSSION TITLE
使用Appveyor构建VS项目-快速编译 | 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#19] [General]
# 使用Appveyor构建VS项目-快速编译 | 倾旋的博客

在工作中经常会看到一些做的比较好的C++开源项目，但是作者没有编写CI/CD去构建项目，发布Release版本的二进制文件，因此需要自己手动编译，但有时候又是临时的环境，还需要安装Visual Studio之类的IDE来构建，比较麻烦，因此appveyor可以支持各类语言的构建环境，只需要在网页上就可以发布二进制程序，解决了本地安装开发环境的痛点。

https://payloads.online/article/9e329caf-052d-4bd9-908b-0ae40a0058bd

**Comment by yangzhao917** [comment]
LGTM
